CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [1.7.5]
```

Comment:
It looks like 1.7.5 was already tagged with just an update to the clj-ldap dependency and this should be 1.7.6?

Reply:
Oh whoops.